### PR TITLE
update helpernode and install-playbook tags

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <prajot.parab@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/212#" title="Author self-approved">Prajyot-Parab</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: variables.tf
Comment (yussufsh) [inline variables.tf:254]: Can we have the latest commit hash?

Comment (Prajyot-Parab) [reply variables.tf:254]: yes will update

Comment (Prajyot-Parab) [reply variables.tf]: Done

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/212#" title="Author self-approved">Prajyot-Parab</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/212#pullrequestreview-578560849" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/212#pullrequestreview-578598598" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/212#" title="Author self-approved">Prajyot-Parab</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/212#pullrequestreview-578560849" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
